projects/perception/lightweight_open_pose/demos/inference_demo.py

```diff
@@ -44,12 +45,13 @@
     # Download one sample image
     pose_estimator.download(path=".", mode="test_data")
     image_path = join("temp", "dataset", "image", "000000000785.jpg")
-    img = cv2.imread(image_path)
+    img = Image.open(image_path)
 
     if onnx:
         pose_estimator.optimize()
 
     poses = pose_estimator.infer(img)
+    img = img.opencv()
```

Comment:
This line overwrites a variable with a new type. Obviously, this is not a bug but it might be a bit confusing.

Reply:
Fixed.